lib/dor/workflow/client/workflow_routes.rb

```diff
@@ -51,6 +51,7 @@ def create_workflow_by_name(druid, workflow_name, version:, lane_id: 'default',
         def update_status(druid:, workflow:, process:, status:, elapsed: 0, lifecycle: nil, note: nil, current_status: nil)
           raise ArgumentError, "Unknown status value #{status}" unless VALID_STATUS.include?(status)
           raise ArgumentError, "Unknown current_status value #{current_status}" if current_status && !VALID_STATUS.include?(current_status)
+          raise ArgumentError, "Do not set status to 'error', use update_error_status" if status == 'error'
```

Comment:
Suggest "use update_error_status method" for a bit more clarity.

Reply:
Good suggestion. Done.